src/frontend/src/views/Data/components/TreeActionMenu/components/OpenInNextstrainButton/index.tsx

```diff
@@ -24,6 +24,7 @@ const OpenInNextstrainButton = ({ item }: Props): JSX.Element => {
   return (
     <>
       <Tooltip
+        role="none"
```

Comment:
This stops the screen reader from putting the Nextstrain button in its own group.  It should be grouped with the rest of the tree actions for each tree.